CSC.md

```diff
@@ -0,0 +1,19 @@
+# 中国学生社区 (Chinese Student Community) 相关内容
+
+## 目录
+
+## CSC 简介
+
+明尼苏达大学（UMN）的中国学生社区（CSC）致力于支持中国国际学生过渡到美国的生活。CSC 成立于 2023 年，它的理念是为 UMN 的中国学生量身定做一个强大的社区，分享宝贵的资源和促进文化理解与适应。
+
+## CSC 微信公众号
+
+> 公众号 ID：gh_970b17d03a2a
+> 公众号二维码：
```

Comment:
Should we link an image here?

Reply:
I'm waiting for the images to be updated so I could include a permanent link here. 

Are there any other ways of doing it?

Reply:
They could be done in the same PR, e.g. you could upload an image to sub dir of /images, then refer the images here like https://www.mingcns.org/images/xxx/yyy.png and it will work after the PR.